azerothcore/azerothcore-wotlk
# fix(Scripts/UBRS): General Drakkisath improvements

* Added spells Rage & Pierce Armor (vMaNGOS)
* Corrected Flamestrike spell ID (vMANGOS)
* He now casts conflagration on a random target



## Changes Proposed:
* Added spells Rage & Pierce Armor (vMaNGOS)
* Corrected Flamestrike spell ID (vMANGOS)
* He now casts conflagration on a random target

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8872

## SOURCE:

vManGOS

## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. go creature id 10363
2. Check his abilities
**3. Don't use cheat god, it bugs conflagrate damage (will always say "absorbed")**

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
